mitosheet/mitosheet/step_performers/import_steps/excel_import.py

```diff
@@ -61,7 +61,7 @@ def execute( # type: ignore
         if not has_headers:
             read_excel_params['header'] = None
 
-        df_dictonary = pd.read_excel(file_name, **read_excel_params) 
+        df_dictonary = pd.read_excel(file_name, **read_excel_params, engine='openpyxl') 
```

Comment:
TODO: remove this. It shouldn't be needed (and we want it to default elsewhere, and give instructions for earlier pandas versions).